tools/files/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    branches:
+      - main
+      - stable-*
+    tags:
+      - "*"
+
+jobs:
+  linters:
+    uses: ./.github/workflows/linters.yml # use the callable linters job to run tests
+  ansible-lint:
+    uses: ansible/ansible-content-actions/.github/workflows/ansible_lint.yaml@main
+  changelog:
+    uses: ansible/ansible-content-actions/.github/workflows/changelog.yaml@main
+  sanity:
+    uses: ansible/ansible-content-actions/.github/workflows/sanity.yaml@main
```

Comment:
I could be wrong but it looks to me like the matrix used by this sanity workflow includes `devel` as a "voting" version which our current workflow does not.

Reply:
I shall check with the devtools team on this.

Reply:
Any updates on this? It's my only remaining concern before approving the PR :)

Reply:
[This block](https://github.com/ansible/cloud-content-ci-automation/pull/1/files#diff-8b14a369e01fa991ac2fa33907ce292d40eddc05ce5792c70c6964740581b6b4R12-R13) will set the tests as non-voting.